Developer-Guide.md

```diff
@@ -227,18 +226,29 @@ $ (cd /usr/share/kata-containers && sudo ln -sf "$image" kata-containers-initrd.
 # Install guest kernel images
 
 ```
-$ sudo ln -s /usr/share/clear-containers/vmlinux.container /usr/share/kata-containers/
-$ sudo ln -s /usr/share/clear-containers/vmlinuz.container /usr/share/kata-containers/
-```
-
-> **Note:**
->
-> - The files in the previous commands are from the Clear Containers
->   `linux-container` package. See [Assumptions](#assumptions).
-
-If you want to use an initrd image, install the [hyperstart](https://github.com/hyperhq/hyperstart/) kernel instead:
-```
-$ sudo wget -O /usr/share/kata-containers/vmlinuz.container https://github.com/hyperhq/hyperstart/raw/master/build/arch/x86_64/kernel
+kernel_arch="$(arch)"
+tmpdir="$(mktemp -d)"
```

Comment:
I know it's a pita, but can you prefix these command lines with `$ ` please? :)

@klynnrif - ptal.